File: problem-1/main.java
class ProblemOne {
    public static void main(String[] args){
        int s = 0;
        for (int i = 0; i < 1000; i++){
            if (i % 5 == 0 || i % 3 == 0){
                s += i;
            }
        }
        System.out.println(s);
    }
}

File: problem-1/main.cs
using System;

namespace ProjectEuler {
    class ProblemOne {
        static void Main(string[] args) {      
            int s = 0;
            for (int i = 0; i < 1000; i++)
            {
                if (i % 3 == 0 || i % 5 == 0){
                    s += i;
                }
            }
            Console.WriteLine(s);
        }
    }
}